packages/charts/react-charts-preview/library/src/components/DonutChart/Arc/Arc.types.ts

```diff
@@ -31,9 +31,9 @@ export interface ArcProps {
   outerRadius: number;
 
   /**
-   * Color for the Arc.
+   * Gradient for the legend in the chart. If not provided, it will fallback on the default color palette.
    */
-  color: string;
+  gradient: [string, string];
```

Comment:
the term gradient is confusing. Lets use color only with this structure -> string | [string, string]

Reply:
I meant in all places.

Reply:
ohh, ok, so it should work with either a single color string or gradient tuple

Reply:
correct.